ams/ams-server/src/test/java/com/netease/arctic/ams/server/AmsTestBase.java

```diff
@@ -194,6 +194,7 @@ public static void beforeAllTest() throws Exception {
     com.netease.arctic.ams.server.config.Configuration configuration =
         new com.netease.arctic.ams.server.config.Configuration();
     configuration.setString(ArcticMetaStoreConf.DB_TYPE, "derby");
+    configuration.setString("arctic.ams.terminal.local.spark.sql.session.timeZone", "UTC");
```

Comment:
Please  repalce the key with  a static variable  like  'ArcticMetaStoreConf.DB_TYPE'

Reply:
@Aireed CC 
I think this property is not a standard configuration for ArcticMetaStore, I add  this property just to test Spark